src/core.ts

```diff
@@ -10,9 +12,10 @@ import getPlugin from "./plugins";
 import { enqueuePayload, flushPayloadQueue, resetUploads, upload } from "./upload";
 import { getCookie, getEventId, guid, isNumber, setCookie } from "./utils";
 
-export const version = "0.2.4";
+export const version = "0.2.5";
 export const ClarityAttribute = "clarity-iid";
 export const InstrumentationEventName = "Instrumentation";
+const MarkEventName = "Mark";
```

Comment:
>Mark [](start = 23, length = 4)

What does a "Mark" event mean? Not obvious from the name.

Reply:
It is custom event for the user at any time in the page's lifecycle. The inspiration is from performance.mark() API, which lets you create custom performance entries in the browser:
[https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)

---
In reply to: [250282778](https://github.com/Microsoft/clarity-js/pull/113#discussion_r250282778) [](ancestors = 250282778)